### PR TITLE
Hello TensorFlow

### PR DESCRIPTION
**Pull request by ghost**
Including examples using java, python and scala (tensorflow_scala from eaplatanios)

### CODE REVIEW

File: examples/haskell/tf/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (RHDZMOTA) [inline examples/haskell/tf/.vscode/settings.json:1]: Maybe we should ignore this file in order to remain "IDE agnostic". What do you think?

File: examples/haskell/tf/ChangeLog.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Changelog for tf
+
+## Unreleased changes
```
Comment (RHDZMOTA) [inline examples/haskell/tf/ChangeLog.md:3]: What is the purpose of this file? Is it relevant for the example?

File: examples/haskell/tf/LICENSE [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Copyright Oscar Vargas Torres (c) 2018
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
```
Comment (RHDZMOTA) [inline examples/haskell/tf/LICENSE:5]: I'm not well aware of the licensing "best practices". But isn't the top-level license enough? Let me know your opinion on this subject.

File: examples/scala/addition/build.sbt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+name := "addition"
+
+version := "0.1"
+
+scalaVersion := "2.12.6"
+
+libraryDependencies ++= Seq(
+  "org.platanios" %% "tensorflow" % "0.2.3",
+  "org.platanios" %% "tensorflow" % "0.2.3" classifier "darwin-cpu-x86_64"
```
Comment (RHDZMOTA) [inline examples/scala/addition/build.sbt:9]: We have to consider if we are going to focus on a single platform (i.e., Mac, Linux) or be platform agnostic. I would suggest being Linux compatible. What do you think?

File: examples/python/addition.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
+
+# Define computational graph
+X = tf.placeholder(tf.float32, name="X")
+Y = tf.placeholder(tf.float32, name="Y")
+
+addition = tf.add(X, Y, name="addition")
+
+
+# Create the session
```
Comment (RHDZMOTA) [inline examples/python/addition.py:14]: Just to follow python's best practices we should consider having a `requirements.txt` and using a `main` function: 

```python
import os
import tensorflow as tf

os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'

def main(): 
    X = tf.placeholder(tf.float32, name="X")
    Y = tf.placeholder(tf.float32, name="Y")
    addition = tf.add(X, Y, name="addition")

    with tf.Session() as session:
        result = session.run(addition, feed_dict={X: [1, 2, 10], Y: [4, 2, 10]})
        print(result)

if __name__ == '__main__':
  main()
``` 

With your comments, of course (:

**Review by RHDZMOTA — COMMENTED** [review]
Thanks a lot for your contributions, Oscar! Just some minor details regarding the directory structure; what do you think about using a standard path for every example? 

This would be `examples/<lang>/hello-tf/<code/project>` for the Hello Tensorflow example and `examples/<lang>/tensor-operations/<code/project>` for the Operations examples (e.g., addition, subtraction, etc).

File: examples/haskell/tf/.vscode/settings.json (hunk 1 above)
Comment (ghost) [reply examples/haskell/tf/.vscode/settings.json:1]: Sounds OK. I'll add .vscode to .gitignore

File: examples/haskell/tf/ChangeLog.md (hunk 2 above)
Comment (ghost) [reply examples/haskell/tf/ChangeLog.md:3]: Good catch. Thanks. I'll remove the file (was created automatically for me by stack)

File: examples/haskell/tf/LICENSE (hunk 3 above)
Comment (ghost) [reply examples/haskell/tf/LICENSE:5]: Same situation: Subproject created by haskell-stack asked for a license. I chose one. But, it is a good point. I'll remove this one too.

File: examples/scala/addition/build.sbt (hunk 4 above)
Comment (ghost) [reply examples/scala/addition/build.sbt:9]: Sure!
That's a very valid concern.

Are you aware of any way to configure it on runtime based on OS information obtained with SBT? That woud be awesome.

File: examples/python/addition.py (hunk 5 above)
Comment (ghost) [reply examples/python/addition.py:14]: 1. `requirements.txt` could be used with pip. What about some other package managers, like conda?
Of course, `requirements.txt` should have `tensorflow` as a dependency.

I need to keep improving this example with documentation and considering different installation/OSes scenarios.
 
2. Sure! I simply used source code that I got from LinkedIn Learning course on TensorFlow.

**Review by RHDZMOTA — COMMENTED** [review]
LGTM

**Pull request merged**